Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
+| Vulnerability scanning              | ✓ Trivy, Clair                                | ✓ Clair                                                             | ✗                            |
+| Content Trust and Validation        | ✓ Cosign                                      | ✓ Cosign                                                            | ✗                            |
+| Multi-tenancy                       | ✓ (not on the storage level)                  | ✓ (not on the storage level)                                        | ✓ (not on the storage level) |
+| Backup and restore                  | ✓                                             | ✓                                                                   | ✗                            |
+| Monitoring                          | ✓ Prometheus metrics                          | ✓ Prometheus metrics                                                | ✓ Prometheus metrics         |
+| HA mode                             | ✓                                             | ✓                                                                   | ✗                            |
+| Registry replication                | ✓                                             | ✓                                                                   | ✓                            |
+| Proxy cache                         | ✓                                             | ✓ Feature is in the technology preview stage (non production ready) | ✗                            |
+| Quota management                    | ✓ Based on storage consumption or image count | ✓ Based on storage consumption                                      | ✗                            |
+| Garbage collection                  | ✓                                             | ✓                                                                   | ✗                            |
+| Retention policy                    | ✓ Multiple tag retention rules                | ✓ Only tag expiration rules                                         | ✗                            |
+| Additional supported artifacts      | ✗ (only OCI artifacts)                        | ✗ (only OCI artifacts)                                              | ✓ Maven, YUM                 |
+| Integration possibilities           | ✓ Dragonfly (P2P), Kraken (P2P)               | ✗                                                                   | ✓ Harbor, Nydus, eStargz     |
+
```

Comment:
IaC capabilities or CRD capability might be relevant as well

Reply:
We mentioned it in the desirable features 2cc9b1b958e968bf3f0cbd23c7380cd8a716427d